debug.out

```diff
@@ -0,0 +1,302 @@
+[
```

Comment:
What is this file?

Reply:
I'm sorry this is an artifact of debugging. Deleted and added to gitignore in the latest commit.